proto/spacemesh/global_state_types.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package spacemesh;
+import "spacemesh/types.proto";
+
+// Data items related to an account
+enum AccountDataFlag {
+    NONE = 0;
+    TRANSACTION_RECIEPT = 1; // tx receipt for a tx to or from an account
+    REWARD = 2; // reward awarded to an account
+}
+
+message AccountDataParams {
+    AccountId account_id = 1;
+    uint32 account_data_bit_field = 2; // A bit field of AccountDataFlag
```

Comment:
This was called `flags` in mesh_types, can we harmonize the naming?

https://github.com/spacemeshos/api/blob/321c19757e5381f56c306f1eb99660ba72f819ae/proto/spacemesh/mesh_types.proto#L14

Reply:
I think we should use descriptive var names so I'll change flags to something better.

Reply:
I've changed all `flag` fields to more descriptive names.